resources/profiles/Creality.json

```diff
@@ -1437,6 +1573,22 @@
         {
             "name": "Creality K1 Max (0.8 nozzle)",
             "sub_path": "machine/Creality K1 Max (0.8 nozzle).json"
+        },
+        {
+            "name": "Creality K2 Plus 0.2 nozzle",
+            "sub_path": "machine/Creality K2 Plus 0.2 nozzle.json"
+        },
+        {
+            "name": "Creality K2 Plus 0.4 nozzle",
+            "sub_path": "machine/Creality K2 Plus 0.4 nozzle.json"
+        },
+        {
+            "name": "Creality K2 Plus 0.6 nozzle",
+            "sub_path": "mahcine/Creality K2 Plus 0.6 nozzle.json"
```

Comment:
typo  mahcine > machine

Reply:
Thank you for that! I need to stop working on these profiles late at night...